packages/shared/src/components/cards/survey/FeedSurveyCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement, useState } from 'react';
+import classNames from 'classnames';
+import { Card as CardV1 } from '../v1/Card';
+import { Card } from '../Card';
+import { useFeedLayout } from '../../../hooks';
+import { Button, ButtonSize, ButtonVariant } from '../../buttons/Button';
+import { RatingStars } from '../../utilities/RatingStars';
+import { useFeedSurvey } from '../../../hooks/feed/useFeedSurvey';
+import { FeedSurveyProps } from './common';
+
+export function FeedSurveyCard({
+  max,
+  title,
+  lowScore,
+  postFeedbackMessage = 'Thank you for your feedback!',
+}: FeedSurveyProps): ReactElement {
+  const [score, setScore] = useState(0);
+  const { submitted, onSubmit, onHide } = useFeedSurvey({ score });
+  const { shouldUseListFeedLayout } = useFeedLayout();
+  const CardComponent = shouldUseListFeedLayout ? CardV1 : Card;
+
+  return (
+    <div
+      className="relative rounded-16 p-px"
+      style={{
+        backgroundImage:
+          'linear-gradient(180deg, rgba(255, 233, 35, 1) 0%, rgba(252, 83, 141, 1) 50%, rgba(113, 71, 237, 1) 100%)',
```

Comment:
Mayvbe we can save these linear gradients somewhere since we're using in both card and banner

Reply:
Yeah, let's do that to make it easier to maintain.

Reply:
Also better to use color-mix on these